step-framework-parent/step-framework-collections/src/main/java/step/core/collections/delegating/DelegatingCollectionFactory.java

```diff
@@ -51,6 +52,16 @@ public <T> Collection<T> getCollection(String name, Class<T> entityClass) {
 		return collectionFactory.getCollection(name, entityClass);
 	}
 
+	@Override
+	public Collection<VersionableEntity> getVersionedCollection(String name) {

```

Comment:
This method could be abstract and is the same for all CollectionFactories

Reply:
I'll take the point as part of the refactoring ticket for next release